### ISSUE TITLE
RT Fireflies are rather common

### ISSUE THREAD

**Issue by ruvaldak** [sultim-t/xash-rt#42]
Noticed that there are a bit more fireflies than expected. Can see them especially when the screen is paused. Seems they tent to occur most around the edges of light sources, but it's rather random, as is the case for RT fireflies. They also stick around for a lot longer than expected - in some cases several seconds if you don't move your camera.

**Comment by sultim-t** [comment]
Yeah, somewhere in the light sampling, the produced radiance is overly large for some of the random numbers

**Comment by ruvaldak** [comment]
Some fireflies are expected with any form of realtime raytracing, just seems a bit worse here than it should be. Not to the point of ruining the image entirely, but enough to be noticeable in almost all scenes - not just dark ones.

**Comment by Speedy-VonGofast** [comment]
Fireflies are worse in many unlit rooms that entirely rely on the sunlight bounces on Surface Tension maps, especially in the supply room that have to be unlocked by a security guard. This room is a firefly sh*tshow, moving the camera around produces a lot of dark spots on the edges of the screen, the viewmodel and the flashlight.
